packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -3,532 +3,645 @@ module.exports = {
     // enforce line breaks after opening and before closing array brackets
     // https://eslint.org/docs/rules/array-bracket-newline
     // TODO: enable? semver-major
-    'array-bracket-newline': ['off', 'consistent'], // object option alternative: { multiline: true, minItems: 3 }
+    "array-bracket-newline": ["off", "consistent"], // object option alternative: { multiline: true, minItems: 3 }
```

Comment:
please revert the unrelated and incorrect formatting (this guide requires JS to use single quotes)

Reply:
ok @ljharb